torchrl/modules/planners/cem.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+
+from torchrl.data.tensordict.tensordict import TensorDictBase
+from torchrl.envs import EnvBase
+from torchrl.modules.planners import MPCPlannerBase
+
+__all__ = ["CEMPlanner"]
+
+
+class CEMPlanner(MPCPlannerBase):
+    """
+    CEMPlanner Module. This class inherits from TensorDictModule.
```

Comment:
this should be 

```
"""CEMPlanner Module. This class inherits from TensorDictModule.

etc
"""
```

Also I would not say in the first line that it inherits from `TensorDictModule`, is it so important that it must be in the headline?

Reply:
Missing ref (if you have one)